cmd/prow-job-dispatcher/main.go

```diff
@@ -661,38 +672,51 @@ func cleanup(directory string) {
 	logrus.WithField("directory", directory).Info("Successfully removed directory")
 }
 
+// createPR creates PR with config changes and sanitizer changes, it causes app to exit in
+// case of failure to trigger re-run of logic
 func createPR(o options, config *dispatcher.Config, pjs map[string]string) {
 	targetDirWithRelease := filepath.Join(o.targetDir, "/release")
 	cleanup(targetDirWithRelease)
 	defer cleanup(targetDirWithRelease)
 
 	logrus.WithField("targetDir", o.targetDir).Info("Changing working directory ...")
 	if err := os.Chdir(o.targetDir); err != nil {
-		logrus.WithError(err).Error("failed to change to root dir")
-		return
+		logrus.WithError(err).Fatal("failed to change to root dir")
```

Comment:
Why do you prefer `logrus.Fatal(...)` over returning an `error`?

Reply:
logging overall and error handling will be reworked in a separate PR (mon or tue). This is fallout after this tool running as a single run tool and I have this on my list.